### ISSUE TITLE
Merge with parquet.net

### ISSUE THREAD

**Issue by aloneguid** [dazfuller/datatable-to-parquet#1]
Would you like to merge this into main parquet project as a separate package? Someone already asked for this.

**Comment by dazfuller** [comment]
Yes, yes I would :) Given the number of times I have to dig this out to do it again, having it as part of the project would make sense

**Comment by aloneguid** [comment]
Awesome :) Here is an active project https://github.com/aloneguid/parquet-dotnet
